substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfLineSectionImpl.java

```diff
@@ -652,15 +593,15 @@ private static SubRange prologueLeafRange(CompiledMethodEntry compiledEntry) {
     }
 
     private int writeCopyOp(DebugContext context, byte[] buffer, int p) {
-        byte opcode = DW_LNS_copy;
+        byte opcode = DW_LNS_copy.value();
         int pos = p;
         debugCopyCount++;
         verboseLog(context, "  [0x%08x] Copy %d", pos, debugCopyCount);
         return writeByte(opcode, buffer, pos);
```

Comment:
Can't we just have a `writeByte(DwarfLineOpcode opcode, ...)` which calls `writeByte(opcode.value(), buffer, pos)`?

Reply:
Done.